### ISSUE TITLE
Site not responsive

### ISSUE THREAD

**Issue by Sarabjeet108** [gbowne1/spmssite#41]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/gbowne1/spmssite/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/gbowne1/spmssite/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/gbowne1/spmssite/issues) that already report this problem, without success.

### Current Behavior

By reducing the size of the browser, the menu bar including "about", "material", "industry" etc buttons get hidden.

### Expected Behavior

the buttons should not disappear and should fit the screen as the browser window size is reduced

### Steps to Reproduce

You can download the zip file of the code and run the index.html file to run the website

**Comment by gbowne1** [comment]
Yes, this is absolutely an issue.  It's repsonsive, but only so much.  I tested this in Chrome and using the mobile/tablet views in Devtools.  When you reduce the page the navbar is quite a mess.

The hamburger is not working all that great either.

**Comment by Sarabjeet108** [comment]
@jlmb54 just fork this project and start working. Once the issue is resolved, create a pull request.

**Comment by jlmb54** [comment]
I started working on the issue and for me the responsiveness of the navbar works quite well. Could you describe me the problem in more detail @Sarabjeet108? I have tested the project in the Chrome browser and Frefox in several screen sizes. When the screen size is less than 1.000px the buttons do disapear, but the burger menu appears and the navigation items are accessible through it.

**Comment by Sarabjeet108** [comment]
@jlmb54 t reduce the browser window and use the burger button. Now change back the browser window to the original size. The burger menu and all the buttons get disappeared.
try with the different window sizes while using the burger button.

**Comment by gbowne1** [comment]
Yeah it does have some issues.  

The navbar needs completely redone anyhow.  I made a seperate issue with the required changes. 

See #38 feat: Capabilities issue

About (about.html) needs a dropdown child of history with using a # link.

Capabilities gets child of services, equipment, material, etc.

Use the Bootstrap 5.2 Navbar.

I forgot the need for light mode/dark mode toggle too. And search field with button, thats mainly only used to search the site.

**Comment by jlmb54** [comment]
Thanks for the quick reply. I will have a look at it.

**Comment by gbowne1** [comment]
My logo is proprietary.. this is a template website.. I would prefer using a image placeholder instead of my logo.  But that favicon is good for a placeholder.

**Comment by gbowne1** [comment]
In my testing the site is responsive on Google Chrome @ 990 - 992 x 600 through max.  Doesn't look very good when under that size.  The responsive hamburger doesn't appear till 990-992 x {random height} .
